ci-operator/config/openshift/release/openshift-release-master__nightly-4.7.yaml

```diff
@@ -284,6 +288,17 @@ tests:
     env:
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-aws
+- as: e2e-aws-upgrade-rollback-first-z
```

Comment:
I think this might need to be in a new variant so the naming looks right.  I don't think `first-z` is relevant.  I would say `e2e-aws-upgrade-rollback-oldest-supported`

Reply:
`e2e-aws-upgrade-rollback-first-z` -> `e2e-aws-upgrade-rollback-oldest-supported` in 9f0e0dec87 -> 7e35f2921d.